components/raftstore/src/store/fsm/apply.rs

```diff
@@ -2791,6 +2803,41 @@ where
         ))
     }
 
+    fn set_flashback_state_persist(
+        &self,
+        ctx: &mut ApplyContext<EK>,
+        req: &AdminRequest,
+    ) -> Result<(AdminResponse, ApplyResult<EK::Snapshot>)> {
+        let region_id = self.region_id();
+        let region_state_key = keys::region_state_key(region_id);
+        let mut old_state: RegionLocalState = match ctx
```

Comment:
Type annotation here is unnecessary.

```suggestion
        let mut old_state = match ctx
```